### PR TITLE
#168285248, #168285305 Rest Endpoint accept and reject

### PR DESCRIPTION
**Pull request by Regiss05**
#### What does this PR do?
accept and deny request

#### Description of Task to be completed?
mentor should accept or deny mentorship request session

#### How should this be manually tested?
- /api/v1/sessions/:sessionId/accept
- /api/v1//sessions/:sessionId/reject

#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/168285305
- https://www.pivotaltracker.com/story/show/168285248

#### Screenshots (if appropriate)
feed end7

#### Questions:

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,5 +1,2 @@
-export default{
-  jwt : {
-    secreKey : 'AppSecretKey1'
-  }
-}
\ No newline at end of file
+
+    secret = cycle10
```
Comment (oesukam) [inline .env:2]: Blank line required and .env file should be ignored, you can’t push it to public

File: .nyc_output/defd535a-6e2d-4f09-8402-eb289b987973.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (oesukam) [inline .nyc_output/defd535a-6e2d-4f09-8402-eb289b987973.json]: blank line required

File: .nyc_output/processinfo/defd535a-6e2d-4f09-8402-eb289b987973.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"uuid":"defd535a-6e2d-4f09-8402-eb289b987973","parent":null,"pid":10388,"argv":["C:\\Program Files\\nodejs\\node.exe","D:\\free-mentors\\node_modules\\mocha\\bin\\mocha","--exit","server/v1/tests/*js"],"execArgv":[],"cwd":"D:\\free-mentors","time":1567686085186,"ppid":19568,"root":"85ea48c0-eb6d-48cd-84b3-425c9eb24638","coverageFilename":"D:\\free-mentors\\.nyc_output\\defd535a-6e2d-4f09-8402-eb289b987973.json","files":[]}
```
Comment (oesukam) [inline .nyc_output/processinfo/defd535a-6e2d-4f09-8402-eb289b987973.json]: This file and the entire folder should be ignored

File: .travic.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+script:
+  - npm test
+  - npm run build
+
+deploy:
+  - provider: pages
+  - skip-cleanup: true
+  - github-token: $GITHUB-TOKEN
+
+  on:
```
Comment (oesukam) [inline .travic.yml]: Blank line required

**Review by oesukam — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (Regiss05) [reply .env:2]: yes I'm going to put it into gitignore

File: .env [hunk 5]
```diff
@@ -1,5 +1,2 @@
-export default{
-  jwt : {
-    secreKey : 'AppSecretKey1'
-  }
-}
\ No newline at end of file
+
```
Comment (oesukam) [inline .env:1]: New line required

File: .nyc_output/processinfo/index.json [hunk 6]
```diff
@@ -0,0 +1 @@
+{"processes":{"322b87ab-99a9-4a78-b3e7-d81904ba9dba":{"parent":null,"children":[]}},"files":{},"externalIds":{}}
```
Comment (oesukam) [inline .nyc_output/processinfo/index.json:1]: This file and folder should be ignore

**Review by oesukam — APPROVED** [review]
Add the screenshots for postman

**Pull request merged**